golem-worker-service-base/src/http/http_request.rs

```diff
@@ -422,7 +193,7 @@ mod tests {
 
         let resolved_route = api_request.resolve(&api_specification).unwrap();
 
-        let result = WorkerRequest::from_resolved_route(resolved_route.clone());
+        let result = resolved_route.worker_request;
```

Comment:
This is an example that shows the workflow is better now. This is because, we are resolving the concrete worker request during the route resolution instead of a 2 step process (i.e, expr that is required to form the worker request gets evaluated as early as possible).

Reply:
Previously, the expr got propogated even further closer to the evaluation stage, but that's not needed.